src/core/class/Base.ts

```
@@ -0,0 +1,133 @@
+      },
+      { ...opts },
+    );
+  }
+
+  async react(emoji: string, message?: WAMessage): Promise<WAMessage | undefined> {
+    return await this.client.sendMessage(this.jid, {
+      react: { text: emoji, key: message?.key || this.key },
+    });
+  }
+
+  async delete(message: WAMessage): Promise<void | WAMessage> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)